TrackerConditions/src/StrawElectronicsMaker.cc

```diff
@@ -16,14 +16,16 @@ namespace mu2e {
 
   StrawElectronics::ptr_t StrawElectronicsMaker::fromFcl(EventTiming::cptr_t eventTiming) {
 
+    unsigned maxTDC = pow(2,_config.numTDCbits());
```

Comment:
**his is my paranoia talking.  In the old days this would have been implemented as

<pre>
   unsigned maxTDC = static_cast<unsigned>(std::round(exp( 24. * log(2.))));
</pre>

Even though 2, 24 and 16777215 are all exactly representable as doubles, the final result could suffer from round-off error.  The the presence or absence of a round off problem could be platform dependent and compiler version dependent.  I see you have printout to verify that it's correct but people won't remember to check it a year from now when we try a new compiler or new hardware.  

Since you only have to do this once, just write the loop.  Or you can find code that will do the minimum multiply algorithm for values known at run time.   There is code in $CETLIB_INC/cetlib/pow.h but it only works of the exponent is known at compile time.

Reply:
Great solution.  I had not thought of that.